src/util/BoxSizing.hxx

```diff
@@ -0,0 +1,101 @@
+#ifndef BOX_SIZING_HXX
+#define BOX_SIZING_HXX
+
+#include "../util/Meta.hxx"
+
+/**
+ * @class       ScreenMeasurement
+ * @brief       Represents a measurement made on the screen
```

Comment:
What measurement? distances in pixels?

Reply:
This is an abstract measurement. It's goal is to measure in pixels at the end of the day. But it keeps representations as window percentage & layout percentage as well